lib/views/layout.erb

```diff
@@ -3,7 +3,9 @@
 <head>
   <link rel="stylesheet" type="text/css" href="<%= url("/css/foundation.css")%>">
   <link rel="stylesheet" type="text/css" href="<%= url("/css/app.css")%>">
-  <script src="<%= url("/js/your-file.css")%>"></script>
+  <script src="<%= url("/js/vendor/foundation.js")%>"></script>
+  <script src="<%= url("/js/vendor/jquery.js")%>"></script>
```

Comment:
Can you make sure this works in the browser? I think you need to include jQuery BEFORE you include foundation.js, but I'm not sure.

Reply:
I'm not sure what it should look like. I'm not getting an error if that's what you mean. :)